tests/vmi_configuration_test.go

```diff
@@ -816,6 +816,32 @@ var _ = Describe("Configurations", func() {
 				}, 10*time.Second)
 			})
 		})
+
+		Context("when CPU features defined", func() {
+			It("should start a Virtaul Machine with matching features", func() {
+				vmiModel := "Conroe"
+				if libvirtCpuVendor == "AMD" {
+					vmiModel = "Opteron_G1"
+				}
+				cpuVmi.Spec.Domain.CPU = &v1.CPU{
+					Model: vmiModel,
+					Features: []v1.CPUFeature{
+						{
+							Name: "apic",
+						},
+						{
+							Name:   "clflush",
+							Policy: "optional",
+						},
+					},
+				}
+
+				By("Starting a VirtualMachineInstance")
```

Comment:
I think this test should be much more expressive.
It needs to show that the VM is scheduled in the right place and has the right flags - or am not understanding something?